GettingStarted.md

```diff
@@ -0,0 +1,45 @@
+# Concepticon
+
+## General information
+
+- https://concepticon.clld.org
+- https://concepticon.clld.org/about
+- https://pdfs.semanticscholar.org/db68/1ae44a635c66b8b2664f0538c93d5982a7c3.pdf
+- a beginner's guide for adding new concept lists: https://calc.hypotheses.org/2225
+- GitHub review workflow: https://calc.hypotheses.org/2680
+- types of concept lists: https://zenodo.org/record/1474751#.XkpNm3VKg5k
+
+## Data and tools:
+
+- authorative source for Concepticon:
+  - https://github.com/concepticon/concepticon-data
+- lists to be linked/developed/etc.:
+  - https://github.com/concepticon/concepticon-dev
```

Comment:
Maybe we should also drop any text about concepticon-dev. As fas as I can tell, we haven't really adopted a workflow where this would play a role.

Reply:
That makes sense since the repository is archived. Will change it accordingly.